canonical/ubuntu.com
# Refresh desktop/developers

**Note: There are separate PRs for the changes to the [D3 graphs ](https://warthogs.atlassian.net/browse/WD-8270)and for the logic needed to allow pagination in the [latest-news module](https://warthogs.atlassian.net/browse/WD-8426)**
## Done

- Applies the [content](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit) and [design](https://www.figma.com/file/wLOtPeQhAiExhQCwKYhi3H/24.04-U.com---desktop-(rebranding)?type=design&node-id=1-2&mode=design&t=Fb5N2muecteluj8U-0) changes

## QA

- Go to https://ubuntu-com-13491.demos.haus/desktop/developers
- Compare against design and copydoc
- Check on different sized screens

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7733
